packages/swapkit/helpers/src/helpers/asset.ts

```diff
@@ -190,24 +193,29 @@ export const getAssetType = ({ chain, symbol }: { chain: Chain; symbol: string }
   if (symbol.includes("/")) return "Synth";
 
   switch (chain) {
+    case Chain.Arbitrum:
+      return [Chain.Ethereum, Chain.Arbitrum].includes(symbol as Chain) ? "Native" : "ARBITRUM";
```

Comment:
ARB is just an ARC-20 no?

Reply:
No xd ARC-20 is sth completely different than ERC20 - we had that convo around AVAX no?😅

Reply:
https://docs.arbitrum.foundation/concepts/arb-token

Reply:
Skipping that classify as spoken - we will only show `Native` for gas assets to not confuse it with governance tokens